dbms/src/Server/Server.cpp

```diff
@@ -1271,9 +1293,6 @@ int Server::main(const std::vector<std::string> & /*args*/)
     if (config().has("macros"))
         global_context->setMacros(std::make_unique<Macros>(config(), "macros"));
 
-    /// Initialize the labels of tiflash compute node.
-    ComputeLabelHolder::instance().init(config());
-
     /// Init TiFlash metrics.
     global_context->initializeTiFlashMetrics();
```

Comment:
It is a bit strange that `TiFlashMetrics` is called before `global_context->initializeTiFlashMetrics()`, how about deleting `Context::initializeTiFlashMetrics` directly?

Reply:
Good idea! I have updated the code.